fdbserver/storageserver.actor.cpp

```diff
@@ -4262,6 +4287,7 @@ ACTOR Future<Void> getMappedKeyValuesQ(StorageServer* data, GetMappedKeyValuesRe
 
 	double duration = g_network->timer() - req.requestTime();
 	data->counters.readLatencySample.addMeasurement(duration);
+	data->counters.mappedRangeSample.addMeasurement(duration);
```

Comment:
This is the same as ReadLatencySample, is this intended?

Reply:
Yes. `readLatencySample` covers all types of read requests, point, range, and mappedRange, while `mappedRangeSample` only tracks mappedRange requests, which is goal of this PR.